mobile/src/screens/marketplace.js

```diff
@@ -323,9 +330,7 @@ class MarketplaceScreen extends Component {
   /* Handle the response from window.onScroll
    */
   handleScrollHandlerResponse = ({ scrollTop }) => {
-    if (scrollTop < -100) {
-      this.dappWebView.injectJavaScript(`document.location.reload()`)
-    }
+    this.setState({ enablePullToRefresh: scrollTop === 0 })
```

Comment:
When I was testing I think everything still worked without having to test for scrollTop = 0

Reply:
I think because the RefreshControl triggers a refresh when scrollY is 0 in the ScrollView it needs to be here. The scrolling happens in the WebView so scrollY is always 0 in the ScrollView. On Android it'll refresh if you try to scroll up after scrolling down in the DApp. I'm not sure why it doesn't do the same thing on iOS though.